core/src/main/scala/scalaz/Enum.scala

```diff
@@ -251,5 +251,13 @@ object Enum {
     }
     z
   }
+
+  import Isomorphism.<=>
+
+  def from[F, G](D: F <=> G)(implicit E: Enum[G]): Enum[F] =
```

Comment:
This should probably be called `fromIso` so it's the same as the other ones

Reply:
Sorry, that's a typo from me, well spotted! Thanks